static/js/main.js

```diff
@@ -0,0 +1,129 @@
+
+function preloadArchi()
+{
+    let darkmode = getCookie('darkmode');
```

Comment:
Use localStorage instead of cookie, the code can be shortened to 

```javascript

const isDarkMode = (localStorage.getItem("theme") === "dark");
if(isDarkMode){
    switchDarrMode('dark');
} else {
    switchDarrMode('main');
}

// or, if more color themes need to be added later

const theme = localStorage.getItem("theme");
if(theme)
    setDarrMode(theme)


```

Reply:
Done 💯